test/markup/asciidoc/default.expect.txt

```diff
@@ -62,4 +62,11 @@ I&#x27;ll join that party, too.
 <span class="hljs-bullet">. </span>one thing (yeah!)
 <span class="hljs-bullet">. </span>two thing <span class="hljs-code">`i can write code`</span>, and <span class="hljs-code">`more`</span> wipee!
 
+<span class="hljs-section">= Document Title (Level 0)</span>
+<span class="hljs-section">== Level 1 Section Title</span>
+<span class="hljs-section">=== Level 2 Section Title</span>
+<span class="hljs-section">==== Level 3 Section Title</span>
+<span class="hljs-section">===== Level 4 Section Title</span>
+<span class="hljs-section">====== Level 5 Section Title</span>
```

Comment:
Can we please add a test case showing the suffix version as well and that it also works?

Reply:
I have added them